src/utils/communitySbt/getIpfsLeaves.ts

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+import { LeafInfo } from '../../entities/communitySbt';
+import { geLeavesIpfsUri } from './helpers';
+
+export async function createLeaves(
+    network: string,
+    seasonId: number,
+): Promise<Array<LeafInfo>> {
+
+    const data = await axios.get(geLeavesIpfsUri(network, seasonId));
+
+    const snaphots : Array<{
+            owner: string
+            badgeType: number,
+            metadataURI: string
+        }> = data.data.snapshot;
+
+    let subgraphSnapshots: Array<LeafInfo> = [];
+
+    for (const entry of snaphots) {
```

Comment:
why not use map here? snaphots.map = ...

Reply:
replaced in c856fd63bdec36d55495e3009171c2af0029d365 :)